pom.xml

```diff
@@ -715,6 +715,42 @@
                 <module>samples/server/petstore/jaxrs/jersey1</module>
             </modules>
         </profile>
+        <profile>
+            <id>jaxrs-spec</id>
+            <activation>
+                <property>
+                    <name>env</name>
+                    <value>java</value>
+                </property>
+            </activation>
+            <modules>
+                <module>samples/server/petstore/jaxrs-spec</module>
+            </modules>
+        </profile>
+        <profile>
+            <id>jaxrs-spec-interface</id>
+            <activation>
+                <property>
+                    <name>env</name>
+                    <value>java</value>
+                </property>
+            </activation>
+            <modules>
+                <module>samples/server/petstore/jaxrs-spec-interface</module>
+            </modules>
+        </profile>
+        <profile>
+            <id>jaxrs-spec-interface-response</id>
+            <activation>
+                <property>
+                    <name>env</name>
+                    <value>java</value>
+                </property>
+            </activation>
+            <modules>
+                <module>samples/server/petstore/jaxrs-spec-interface-response</module>
+            </modules>
+        </profile>
```

Comment:
@fujigon Thanks for including jaxrs-spec samples  in the Travis CI.

We'll move these tests to CircleCI instead as we've setup both JDK7 and JDK8 to test the change in parallel.